packages/suite/src/utils/wallet/coinjoinUtils.ts

```diff
@@ -151,7 +153,11 @@ export const getRegisterAccountParams = (
 export const getMaxRounds = (targetAnonymity: number, anonymitySet: AnonymitySet) => {
     // fallback to 1 if any value is undefined or the object is empty
     const lowestAnonymity = Math.min(...(Object.values(anonymitySet).map(item => item ?? 1) || 1));
-    return Math.ceil((targetAnonymity - lowestAnonymity) / ESTIMATED_ANONYMITY_GAINED_PER_ROUND);
+    const estimatedRoundsCount = Math.ceil(
```

Comment:
lets add some tests to this util

Reply:
6083f69c3668c3b8aa12a36fc35e93a40963bc14